doc/getting_started/index.rst

```diff
@@ -137,7 +137,15 @@ We'll also install Zephyr's multi-purpose west tool.
 
          .. code-block:: bash
 
-            brew install cmake ninja gperf ccache dfu-util qemu dtc python3
+            brew install cmake ninja gperf ccache dfu-util qemu dtc python3 libgit2
```

Comment:
what is libgit2 and why is it required?

Reply:
I think this might be a mistake. We use a python wrapper for this library downstream in NCS. I don't think it's required for vanilla zephyr.

Reply:
When I and Rediar Martin were testing this on Mac Catalina, then we could not proceed without installing libgit2. Not sure why is this a new requirement only on this OS, but it was needed.

Reply:
Haven't tested this on the latest tool manager though

Reply:
This sounds like a bug that needs fixing somewhere. It should not be a requirement.

Can you please share details of the errors you saw so we can figure out what is wrong?

Reply:
@aryannordic please use this new account (@mbolivar-nordic ) for any replies.